src/app/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { TaskList } from 'pages/TaskList';
 import { NavBar } from 'widgets/NavBar/NavBar';
+import { Highcharts } from 'pages/Highcharts';
```

Comment:
unused imports `TaskList` and `Highcharts`

Reply:
Fixed